tests/test_memvid.py

```
@@ -0,0 +1,115 @@
+    # Setup
+    logger = setup_logging()
+    api_keys = get_api_keys()
+    failures = []
+    
+    try:
+        # Initialize Memvid store with configurable rebuild flag
+        memvid_retriever, documents = initialize_memvid_store(force_rebuild=force_rebuild_flag)
+        
+        if memvid_retriever and documents:
+            # Test with predefined queries
+            test_queries = [DIFFICULT_CUSTOMERS_QUERY, ROUGH_DAY_QUERY] + memvid_queries
```

Ruff (0.13.1)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)